CHANGELOG.md

```diff
@@ -2,6 +2,57 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.4.0](sonata-project/SonataDoctrineMongoDBAdminBundle/compare/3.3.0...3.4.0) - 2020-09-24
+### Added
+- [[#353](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/353)] Allow `_sort_by` filter to not be initially defined ([@VincentLanglet](https://github.com/VincentLanglet))
+- [[#420](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/420)] Added returning the own instance in `ProxyQuery` based on PHPDoc of `ProxyQueryInterface` ([@franmomu](https://github.com/franmomu))
+- [[#408](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/408)] Added support for symfony/options-resolver:^5.1 ([@franmomu](https://github.com/franmomu))
+- [[#408](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/408)] Added support for twig 3 ([@franmomu](https://github.com/franmomu))
+- [[#409](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/409)] Added doctrine/persistence as a dependency ([@franmomu](https://github.com/franmomu))
+- [[#379](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/379)] Added `ModelManager::getDefaultPerPageOptions()` ([@franmomu](https://github.com/franmomu))
+- [[#379](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/379)] Added `FieldDescription::getTargetModel()` ([@franmomu](https://github.com/franmomu))
+- [[#378](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/378)] Added `doctrine/collections` dependency ([@franmomu](https://github.com/franmomu))
+- [[#375](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/375)] Added `sonata-project/form-extensions` dependency ([@franmomu](https://github.com/franmomu))
+- [[#346](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/346)] "sonata.admin.manager" tag to "sonata.admin.manager.doctrine_mongodb" service ([@phansys](https://github.com/phansys))
+
+### Changed
+- [[#380](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/380)] Changed `DatagridBuilder` constructor's first parameter typehint from `Symfony\Component\Form\FormFactory` to ([@franmomu](https://github.com/franmomu))
+- [[#380](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/380)] `Symfony\Component\Form\FormFactoryInterface` ([@franmomu](https://github.com/franmomu))
```

Comment:
Looks like this line should be in the above one.

Reply:
Why?

Reply:
Look at https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/380, there only one line.

Reply:
I just copied the output provided by @OskarStark.
You mean lines 19 and 20 should be flipped or merged?

Reply:
Sorry, I meant merged.

Reply:
Why? AFAIK, each line is generated by the items in the changelog sections. Look at #379 by instance, which introduces 2 additions.

Reply:
The problem is that in https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/380 there is only one line, and here there are two.

Reply:
Oh, I see. Your changelog only has a single line and dev-kit splitted it:

![image](https://user-images.githubusercontent.com/1231441/94131418-43f36300-fe34-11ea-9310-a9b817cd3ce6.png)

Ping @OskarStark.